apiserver/organization/create_test.go

```diff
@@ -109,12 +109,13 @@ func TestOrganizationStorage_Create(t *testing.T) {
 		t.Run(n, func(t *testing.T) {
 			ctrl := gomock.NewController(t)
 			defer ctrl.Finish()
-			os, mnp, mauth := newMockedOrganizationStorage(ctrl)
```

Comment:
I left the authorizer tests be to prove the new wrapper correctly implements auth. Should be removed in a follow-up PR as they're now duplicated.